src/plugins/saved_objects_management/public/management_section/mount_section.tsx

```diff
@@ -54,39 +56,42 @@ export const mountManagementSection = async ({ core, mountParams }: MountParams)
   };
 
   ReactDOM.render(
-    <I18nProvider>
-      <Router history={history}>
-        <Switch>
-          <Route path={'/:type/:id'} exact={true}>
-            <RedirectToHomeIfUnauthorized>
-              <Suspense fallback={<EuiLoadingSpinner />}>
-                <SavedObjectsEditionPage
-                  coreStart={coreStart}
-                  setBreadcrumbs={setBreadcrumbs}
-                  history={history}
-                />
-              </Suspense>
-            </RedirectToHomeIfUnauthorized>
-          </Route>
-          <Route path={'/'} exact={false}>
-            <RedirectToHomeIfUnauthorized>
-              <Suspense fallback={<EuiLoadingSpinner />}>
-                <SavedObjectsTablePage
-                  coreStart={coreStart}
-                  taggingApi={savedObjectsTaggingOss?.getTaggingApi()}
-                  spacesApi={spacesApi}
-                  dataStart={data}
-                  actionRegistry={pluginStart.actions}
-                  columnRegistry={pluginStart.columns}
-                  allowedTypes={allowedObjectTypes}
-                  setBreadcrumbs={setBreadcrumbs}
-                />
-              </Suspense>
-            </RedirectToHomeIfUnauthorized>
-          </Route>
-        </Switch>
-      </Router>
-    </I18nProvider>,
+    wrapWithTheme(
+      <I18nProvider>
+        <Router history={history}>
+          <Switch>
```

Comment:
This is just an example on how to use `wrapWithTheme` / `KibanaThemeProvider` when mounting a react app.